sklearn/ensemble/bagging.py

```diff
@@ -281,7 +283,7 @@ def _fit(self, X, y, max_samples=None, max_depth=None, sample_weight=None):
         random_state = check_random_state(self.random_state)
 
         # Convert data
-        X, y = check_X_y(X, y, ['csr', 'csc'])
+        X, y = check_X_y(X, y, ['csr', 'csc'], multi_output=self.multi_output)
```

Comment:
Ya, @johannah I don't see why you you'd rather not have the `multi_output=True` in that func call.